samples/drivers/i2s/i2s_codec/src/main.c

```diff
@@ -0,0 +1,219 @@
+/*
+ * Copyright (c) 2024 NXP
+ *
```

Comment:
Fix copyright here also. No `(c)` in the NXP recommended way of adding copyright. Just use `Copyright NXP 2024`